README.md

```diff
@@ -4,6 +4,9 @@
 
 Consul Terraform Sync (just Sync from here on) is a service-oriented tool for managing network infrastructure near real-time. Sync runs as a daemon and integrates the network topology maintained by your Consul cluster with your network infrastructure to dynamically secure and connect services.
 
+* Website: https://www.consul.io/docs/nia
+* Forum: [Discuss](https://discuss.hashicorp.com/tags/c/consul/29/consul-terraform-sync)
```

Comment:
Going to this URL gives me this.... "Oops! That page doesn’t exist or is private."

Reply:
The Discuss link.

Reply:
yeah the tag needs to be created

Reply:
tag won't be created for a bit, removing this to be merged